packages/cadl-ranch-specs/cadl-ranch-summary.md

```diff
@@ -4747,6 +4747,46 @@ Expected input body:
 {"property": 2022-08-26T18:38:00Z}
 ```
 
+### Type_Property_ValueTypes_Decimal_get
+
+- Endpoint: `get /type/property/value-types/decimal`
+
+Expected response body:
+
+```json
+{ "property": 0.33333 }
+```
+
+### Type_Property_ValueTypes_Decimal_put
+
+- Endpoint: `put /type/property/value-types/decimal`
+
+Expected input body:
+
+```json
+{ "property": 0.33333 }
+```
+
+### Type_Property_ValueTypes_Decimal128_get
+
+- Endpoint: `get /type/property/value-types/decimal128`
+
+Expected response body:
+
+```json
+{ "property": 0.33333 }
+```
+
+### Type_Property_ValueTypes_Decimal128_put
+
+- Endpoint: `put /type/property/value-types/decimal128`
+
+Expected input body:
+
+```json
+{ "property": 0.33333 }
```

Comment:
Don't know much about decimal. Could we test the boundary value?

Reply:
we don't really have any test like that but maybe we could have something like

```tsp
op getDecimals(): { a: decimal128, b: decimal128, c: decimal128 };
op verifySum(@body sum: decimal128): void;
```

and the client test need to call `getDecimals()` then do the sum and upload it back to `verifySum`. If you are not using decimal and float, doubles you would send back the wrong value.

For example in JS(with double) if we sum `0.1` 3 times then you don't end up with `0.3` exactly
<img width="179" alt="image" src="https://github.com/Azure/cadl-ranch/assets/1031227/47825781-c44d-43f7-9004-58cf5992f7dd">

Reply:
Added.